src/main/java/io/confluent/connect/hdfs/HdfsSinkConnectorConfig.java

```diff
@@ -148,6 +148,16 @@
   private static final long ROTATE_INTERVAL_MS_DEFAULT = -1L;
   private static final String ROTATE_INTERVAL_MS_DISPLAY = "Rotate Interval (ms)";
 
+  public static final String ROTATE_SCHEDULE_INTERVAL_MS_CONFIG = "rotate.schedule.interval.ms";
+  private static final String ROTATE_SCHEDULE_INTERVAL_MS_DOC =
+          "The time interval in milliseconds to periodically invoke file commits. This configuration ensures that "
+          + "file commits are invoked every configured interval. Time of commit will be adjusted to 00:00 of selected timezone. "
```

Comment:
Time of commit will be adjusted to 00:00 is a bit confused. Do you mean that the commit time is rounded to the starting time of the next period?